Firestore/Source/API/converters.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_API_OBJC_CONVERTERS_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_API_OBJC_CONVERTERS_H_
```

Comment:
This define doesn't seem to match the actual path...  I'm also not 100% sure if this is meant to be used from C++ or not.  If not, can we assume it'll always be `#import`ed and drop the include guard?  (none of the other files in this directory have an include guard)

Reply:
This value reflects the place where I originally put it, but I didn't like it there because it inverted the dependencies in the CMake build. Unfortunately, in the Objective-C part of the sources, we have linting of header guards disabled so the linter didn't catch this.

Fixed.